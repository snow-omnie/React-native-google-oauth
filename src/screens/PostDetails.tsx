import { View, Text, StyleSheet, Dimensions } from 'react-native'
import React, { useEffect } from 'react'
// import MapBox from '../components/MapBox'
import Biometric from '../components/Biometric'

const PostDetails = () => {


    return (
        <View>
            <Text>PostDetails mapbox</Text>
            {/* <MapBox /> */}
            <Biometric />
        </View>
    )
}

export default PostDetails
const styles = StyleSheet.create({
    page: {
        flex: 1,
        justifyContent: 'center',
        alignItems: 'center',
    },
    container: {
        height: 300,
        width: 300,
    },
    map: {
        flex: 1,

    }
});