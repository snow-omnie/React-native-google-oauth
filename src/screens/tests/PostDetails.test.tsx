import 'react-native'
import React from 'react'
import PostDetails from '../PostDetails'
import renderer from 'react-test-renderer'

test('Post details snapShot', () => {
    const snap = renderer.create(<PostDetails />).toJSON();

    expect(snap).toMatchSnapshot()
})