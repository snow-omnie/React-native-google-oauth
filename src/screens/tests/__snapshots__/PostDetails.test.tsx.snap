// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Post details snapShot 1`] = `
<View>
  <Text>
    PostDetails mapbox
  </Text>
  <View>
    <Text>
      Biometric
    </Text>
  </View>
</View>
`;
